src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_BOUND = 4;
+    private static final int EXAMPLE_BOUND = 100;
+    private static final int CIRCLE_RADIUS = 10;
+    private final Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_BOUND);
+        String color = colorSupplier.getRandomColor();
+        switch (figureNumber) {
+            case 0:
+                return new Square(color, random.nextInt(EXAMPLE_BOUND));

```

Comment:
```suggestion
                return getRandomSquare();
```
lets add private methods to get each random figure - this would make your code easier to read